tests/UI/pages/FO/contactUs/index.js

```diff
@@ -0,0 +1,40 @@
+require('module-alias/register');
+const FOBasePage = require('@pages/FO/FObasePage');
+
+class ContactUs extends FOBasePage {
+  constructor() {
+    super();
+
+    this.pageTitle = 'Contact us';
+    this.validationMessage = 'Your message has been successfully sent to our team.';
+
+    // Selectors
+    this.subjectSelect = '#content select[name=\'id_contact\']';
+    this.emailAddressInput = '#content input[name=\'from\']';
+    this.attachmentLabel = '#filestyle-0';
+    this.messageTextarea = '#content textarea[name=\'message\']';
+    this.sendButton = '#content input[name=\'submitMessage\']';
+    this.alertSuccessDiv = '#content div.alert-success';
+  }
+
+  /*
+  Methods
+   */
+  /**
+   * Send message
+   * @param page
+   * @param contactUsData
+   * @returns {Promise<string>}
+   */
+  async sendMessage(page, contactUsData) {
+    await this.selectByVisibleText(page, this.subjectSelect, contactUsData.subject);
+    await this.setValue(page, this.emailAddressInput, contactUsData.emailAddress);
+    await this.uploadFile(page, this.attachmentLabel, `${contactUsData.fileName}.jpg`);
+    await this.setValue(page, this.messageTextarea, contactUsData.message);
+    await page.click(this.sendButton);
+
+    return this.getTextContent(page, this.alertSuccessDiv);
```

Comment:
This Could be tested with another file type (txt, pdf ...), and without file too.
So I think you could do this instead.

```suggestion
  async sendMessage(page, contactUsData, filePath = null) {
    await this.selectByVisibleText(page, this.subjectSelect, contactUsData.subject);
    await this.setValue(page, this.emailAddressInput, contactUsData.emailAddress);
    
    if(filePath) {
      await this.uploadFile(page, this.attachmentLabel, filePath);
    }
    
    await this.setValue(page, this.messageTextarea, contactUsData.message);
    await page.click(this.sendButton);

    return this.getTextContent(page, this.alertSuccessDiv);
```

Then you can create any type of file you want in the test or testing it without a file.

Reply:
Done thx :+1: